datadog/resource_datadog_monitor_test.go

```diff
@@ -318,11 +328,13 @@ func TestAccDatadogMonitor_UpdatedToRemoveTags(t *testing.T) {
 						"datadog_monitor.foo", "require_full_window", "true"),
 					resource.TestCheckResourceAttr(
 						"datadog_monitor.foo", "locked", "false"),
-					// Tags are sorted
+					// Tags are a TypeSet => use a weird way to access members by their hash
 					resource.TestCheckResourceAttr(
-						"datadog_monitor.foo", "tags.0", "baz"),
+						"datadog_monitor.foo", "tags.#", "2"),
 					resource.TestCheckResourceAttr(
-						"datadog_monitor.foo", "tags.1", "foo:bar"),
+						"datadog_monitor.foo", "tags.2644851163", "baz"),
```

Comment:
Not sure I totally follow where these hashes come from?

Reply:
TF `TypeSet` is internally represented as a map that maps computed hashes to actual values. Since the hashes are always the same for one value, this is the way to get them. I'll amend the comment to explain this.

Reply:
Comment added.